thorlcr/graph/thgraphmaster.cpp

```diff
@@ -1833,7 +1833,8 @@ void CJobMaster::saveSpills()
 
 bool CJobMaster::go()
 {
-    class CWorkunitPauseHandler : public CInterface, implements IWorkUnitSubscriber
+    // detected abort conditions or pause state changes.
+    class CWorkunitStateChangeHandler : public CInterface, implements IWorkUnitSubscriber
```

Comment:
NB: CWorkunitPauseHandler -> CWorkunitStateChangeHandler, rename change only, since it does more than just Pause (also does abort)